src/test/java/bogdanSandu/tests/tema8/pageObjects/RedirectsPage.java

```diff
@@ -0,0 +1,72 @@
+package bogdanSandu.tests.tema8.pageObjects;
+
+import bogdanSandu.tests.tema8.tests.BaseTest;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+import static org.testng.Assert.assertEquals;
+
+public class RedirectsPage extends BaseTest {
+
+    private WebDriver driver;
+
+    String expectedURL = "https://testpages.herokuapp.com/styled/javascript-redirect-test.html";
+
+    public RedirectsPage(ChromeDriver driver){
+        this.driver = driver;
+        PageFactory.initElements(driver,this);
+    }
+
+    public static final String START_PAGE_URL = "https://testpages.herokuapp.com/styled/javascript-redirect-test.html";
+
+    public static final String DELAY_BOUNCE = "delaygotobounce";
+
+    public static final String BACK_BUTTON = "goback";
+
+    public static final String DELAY_BASIC = "delaygotobasic";
+
+    @FindBy(id= DELAY_BASIC)
+    private WebElement delayBasic;
+
+    @FindBy(id= BACK_BUTTON)
+    private WebElement backButton;
+
+    @FindBy(id= DELAY_BOUNCE)
+    private WebElement delayBounceBtn;
+
+
+    public WebElement getBackButton() {
+        return backButton;
+    }
+
+    public WebElement getDelayBounceButton() {
+        return delayBounceBtn;
+    }
+
+    public void goToPage() {
+        driver.get(START_PAGE_URL);
+    }
+
+    public void clickDelayBasicButton() {
+        delayBasic.click();
+        fluentWait((ChromeDriver) driver).until(ExpectedConditions.visibilityOfElementLocated(By.id(BACK_BUTTON)));
```

Comment:
Declare the and initialise the fluentWait either in this page constructor or in a separated WaitUtils class inside utils package.